### PR TITLE
Upgrade graphql-java and extended-scalars

### PR DESCRIPTION
**Pull request by dondonz**
Upgrade to the latest graphql-java and extended-scalars

### CODE REVIEW

File: test/build.gradle.kts [hunk 1]
```diff
@@ -7,15 +7,14 @@ plugins {
...
-    testImplementation("com.graphql-java:graphql-java-extended-scalars:2021-06-29T01-19-32-8e19827") {
+    testImplementation("com.graphql-java:graphql-java-extended-scalars:18.1") {
         exclude(group = "com.graphql-java", module = "com.graphql-java")
     }
     testImplementation("org.reflections:reflections:0.9.12")
     testImplementation("com.fasterxml.jackson.dataformat:jackson-dataformat-yaml:2.9.6")
     testImplementation("com.fasterxml.jackson.module:jackson-module-kotlin:2.9.6")
     testImplementation("io.kotest:kotest-runner-junit5:5.1.0")
     testImplementation("io.strikt:strikt-jvm:0.31.0")
-    testImplementation("com.graphql-java:graphql-java-extended-scalars:2021-06-29T01-19-32-8e19827")
```
Comment (dondonz) [inline test/build.gradle.kts:18]: Removed duplicate

**Comment by dondonz** [comment]
`two-transforms-on-a-field` test is failing. I'll ask about this next week

**Review by bbakerman — APPROVED** [review]

**Review by bbakerman — APPROVED** [review]

File: test/src/test/kotlin/graphql/nadel/tests/hooks/chained-instrumentation-works-as-expected.kt [hunk 2]
```diff
@@ -53,7 +53,7 @@ class `chained-instrumentation-works-as-expected` : EngineTestHook {
                         val document = Parser.parse(query)
                         AstPrinter.printAstCompact(document)
                     }
-                    .isEqualTo("query OpName {hello {name} hello {id}}")
+                    .isEqualTo("query OpName{hello{name}hello{id}}")
```
Comment (gnawf) [inline test/src/test/kotlin/graphql/nadel/tests/hooks/chained-instrumentation-works-as-expected.kt:56]: Glad this is the only place that needs to be updated

**Review by gnawf — APPROVED** [review]

File: test/src/test/resources/fixtures/hidden/cannot-query-hidden-fields.yml [hunk 3]
```diff
@@ -51,7 +51,7 @@ response: |-
     "data": null,
     "errors": [
       {
-        "message": "Validation error of type FieldUndefined: Field 'area51' in type 'World' is undefined @ 'hello/area51'",
+        "message": "Validation error (FieldUndefined@[hello/area51]) : Field 'area51' in type 'World' is undefined",
```
[the remark this answers is not in the document]
Comment (dondonz) [reply test/src/test/resources/fixtures/hidden/cannot-query-hidden-fields.yml:54]: The validation errors were standardised and the location got chucked in at the start, to make translations easier

**Pull request merged**